viewer/viewer/serializers.py

```diff
@@ -1,3 +1,5 @@
+import re
```

Comment:
This looks strange to me. Formatting mistake or import mistake?

Reply:
This is the Python regular expression module: https://docs.python.org/3/library/re.html